src/app/pages/networks/network-detail-view/network-detail-view.component.css

```diff
@@ -0,0 +1,24 @@
+.label
```

Comment:
I know this is me probably being picky but can we try and externalise this to the styles-additional.css?